gorm/collection_operators.go

```diff
@@ -16,7 +16,7 @@ type SortingCriteriaConverter interface {
 }
 
 type FieldSelectionConverter interface {
-	FieldSelectionToGorm(ctx context.Context, fs *query.FieldSelection, obj interface{}) ([]string, error)
+	FieldSelectionToGorm(ctx context.Context, fs *query.FieldSelection, obj interface{}, ignoreCase ...bool) ([]string, error)
```

Comment:
this needs to be a option struct{} not a boolean. consider the meaning is for multiple options.